### PR TITLE
Test case issue fix in HorizontalbarchartWithAxis

### PR DESCRIPTION
**Pull request by v-baambati**
Skipping one test cases which is causing issue in Master branch. we will raise new PR once issue got fixed.

### CODE REVIEW

**Comment by AtishayMsft** [comment]
Fixes #30216

**Comment by srmukher** [comment]
Add a description for the PR.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c1324de387982657a207fe2a331063a526fb7fc8'>c1324de387982657a207fe2a331063a526fb7fc8</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxisRTL.test.tsx [hunk 1]
```diff
@@ -315,7 +315,9 @@ describe('Horizontal bar chart with axis - Subcomponent Labels', () => {
       expect(getById(container, /showDots/i)[0]!.textContent!).toEqual('Stri...');
```
Comment (fabricteam) [inline packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxisRTL.test.tsx:315]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30218&commitId=acdb183f4c8073ba0a0565c1014d6b6f6fa583b9&env=prod&clientType=FLUENTUI)


  <details><summary><b>PeoplePicker</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal.suggestions.chromium.png|12| Changed|
|PeoplePicker.Normal Overflow selected.default.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected.default.chromium.png|0| Removed|
|PeoplePicker.Normal selected.suggestions.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (AtishayMsft) [reply packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxisRTL.test.tsx:315]: Unrelated change.

**Pull request merged**